src/main/java/cheese/command/MarkCommand.java

```diff
@@ -0,0 +1,40 @@
+package cheese.command;
+
+import cheese.CheeseException;
+import cheese.Storage;
+import cheese.TaskList;
+import cheese.Ui;
+import cheese.task.Task;
+
+/**
+ * Command to mark a task as done
+ */
+public class MarkCommand extends UpdateCommand {
+    private final boolean done;
```

Comment:
Similar comment, would be better to rename it to `isDone` to follow the coding standard.